Doc (#1450)
silx-kit/pyFAI: doc/source/index.rst

Fast Azimuthal Integration using Python
=======================================

> PyFAI is a python libary for azimuthal integration of X-ray/neutron/electron scattering data acquired with area-detectors. 

Reviewer: ```suggestion
PyFAI is a python libary for azimuthal integration of X-ray/neutron/electron scattering data acquired with area detectors. 
```

For this, images needs to be re-binned in polar coordinate systems. 

> Additional tools are provided to calibrate the experimental setup, i.e. define where the detector is positionned in space considering the sample and the incident beam. 

Reviewer: ```suggestion
Additional tools are provided to calibrate the experimental setup, i.e. define where the detector is positioned in space considering the sample and the incident beam. 
```

.. figure:: img/overview.png
   :align: center
   :alt: PyFAI is about regridding image in polar space.

> The core idea is to re-distribute the signal acquired with the experimental geometry 

Reviewer: ```suggestion
The core idea is to redistribute the signal acquired with the experimental geometry 
```

> into a geometry suitable for further analysis like Rietveld refinement for power data 

Reviewer: ```suggestion
into a geometry suitable for further analysis, like Rietveld refinement for power data 
```

or Inverse Fourier Transform for SAXS data.    

> Unlike interpolation, this re-distribution conserves the signal, its variance and 

Reviewer: ```suggestion
Unlike interpolation, this redistribution conserves the signal, its variance and 
```

> can used for other type of transformation like distortion correction.  

Reviewer: ```suggestion
can used for other types of transformation like distortion correction.  
```

> Since, the alignement of the beam, of the sample and of the detector can never be perfect,

Reviewer: ```suggestion
Since the alignment of the beam, the sample and the detector can never be perfect,
```

pyFAI tries to cope with it by calibrating their relative position using a 
reference sample material (called calibrant). 
After calibration, the geometry can be saved in a *poni-file* and used to perform azimuthal averaging
of several samples. 

> The geometry used by pyFAi is described in this scheme:

Reviewer: ```suggestion
The geometry used by pyFAI is described in this scheme:
```

Also, you should decide whether it's always pyFAI, always PyFAI, or PyFAI only at the start of a sentence, because I think I've found examples of all 3 styles. :wink:

     
.. figure:: img/PONI.png
   :align: center

>    :alt: The geometry used by pyFAI is inspired by SPD

Reviewer: There's quite a bit of text in this image, which may be important for understanding geometry & PONI files. A user with a screenreader probably won't get any of that, just the alt text. It might be worth either having more detailed alt text, or a caption with a link - "See $SECTION for more on how geometry is represented"

Author: I improved the description... but a sketch is worth 1000 words !

 

This documentation starts with a general descriptions of the pyFAI library.
This first chapter contains an introduction to pyFAI, what it is, what it aims at
and how it works (from the scientists' point of view).
Especially, geometry, calibration, azimuthal integration algorithms are described
and pixel splitting schemes are explained there. 

> Follows tutorials, manual pages of applications, the description of the programming interface 

Reviewer: ```suggestion
There follow tutorials, manual pages of applications, the description of the programming interface 
```

"There follow" is quite an old-fashioned, stuffy phrase, but I can't think of a better alternative that fits in the same place. I'd just ditch this sentence entirely, because it's listing the contents right before the table of contents, which seems redundant.

Author: I re-phrased this part...
